src/content/quickstart/openshift/_index.md

```diff
@@ -120,14 +120,18 @@ Please note that the prep_for_subm.sh script has a few pre-requirements, you wil
 
 {{< subctl-install >}}
 
-### Use cluster-a as broker
+### Install Submariner
+
+To install Submariner in the clusters follow the below steps,
```

Comment:
```suggestion
To install Submariner in the clusters follow the steps below.
```

Reply:
This change was removed when the service discovery is a new page.